src/main/scala/chisel3/util/Arbiter.scala

```diff
@@ -1,149 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
you don't want to delete this.

Do you want some more help on cleaning up this git issue?
If so, please tell me what you see when you do 

```
git remote -v
```
in your chisel3 checkout

Reply:
~~~
$ git remote -v
origin  https://github.com/Shorla/chisel3.git (fetch)
origin  https://github.com/Shorla/chisel3.git (push)
upstream        https://github.com/chipsalliance/chisel3.git (fetch)
upstream        https://github.com/chipsalliance/chisel3.git (push)
~~~
@mwachs5 that is what I got.

Reply:
I would need help cleaning the git issue. I don't want to make any mistake that might cost damage.

Reply:
Thanks for your remote info:

```
// Make sure you have all the latest changes
git fetch upstream
git fetch origin
// get onto the branch for this PR
git checkout origin/master
// Get the correct versions of the files you don't want to change
git checkout upstream/master src/main/scala/chisel3/util/Arbiter.scala
git checkout upstream/master <other files that you don't want to change in this PR>
git add -u // Add the files you just checked out
git commit -m "Revert unnecessary changes"
// Push to this PR branch
git push origin/master
```

Reply:
~~~
$ git push origin/master
fatal: You are not currently on a branch.
To push the history leading to the current (detached HEAD)
state now, use

    git push origin/master HEAD:<name-of-remote-branch>
~~~
at the last stage, it kept saying this.

Reply:
`git` is pretty good about telling you what to do. You can do:

It's saying you need to be on a branch. The way to create a new branch is:
`git checkout -b <branch name>`

Your PR is being made from a branch called `master` which is fairly unusual, normally we would make some feature branch like:

`git checkout -b better-connect-documentation`

But it's hard to change the branch name once you start the PR, so in your case you woudl do:

`git checkout -b master`
`git push origin master` <-- note that there is a space not a slash, my previous instructions were wrong

Reply:
It was successful. Is it all fine now? Can I move on to making other adjustments to the document?

Reply:
@mwachs5 I am still seeing the files changed to be 6.... Should I be worried about that still?

Reply:
Hello good morning @Shorla, yea I think there are somethings that should be reverted. If you allow, I would like to suggest the changes.

Reply:
@Burnleydev1 go ahead